x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -0,0 +1,193 @@
+  jobIds = 'jobIds',
+  latestTimestamp = 'latest_timestamp',
+  docsProcessed = 'docs_processed',
+}
+
+interface Props {
+  items: Groups;
+  statsBarData: JobStatsBarStats;
+  jobsList: Jobs;
+}
+
+export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData }) => {
+  const groupsList = Object.values(items);
+  const [pageIndex, setPageIndex] = useState(0);
+  const [pageSize, setPageSize] = useState(10);
+
+  const [sortField, setSortField] = useState<string>(AnomalyDetectionListColumns.id);
+  const [sortDirection, setSortDirection] = useState<SortDirection>(SORT_DIRECTION.ASC);
+
+  // columns: group, max anomaly, jobs in group, latest timestamp, docs processed, action to explorer
+  const columns: any[] = [
+    {
+      field: AnomalyDetectionListColumns.id,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableId', {
+        defaultMessage: 'Group ID',
+      }),
+      render: (id: Group['id']) => <JobSelectorBadge id={id} isGroup={id !== 'ungrouped'} />,
+      sortable: true,
+      truncateText: true,
+      width: '20%',
+    },
+    {
+      field: AnomalyDetectionListColumns.maxAnomalyScore,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableMaxScore', {
+        defaultMessage: 'Max anomaly score',
+      }),
+      sortable: true,
+      render: (score: Group['max_anomaly_score']) => {
+        if (score === null) {
+          return <EuiLoadingSpinner />;
+        } else {
+          const color: string = getSeverityColor(score);
+          return (
+            // @ts-ignore
+            <EuiHealth color={color} compressed="true">
+              {score >= 1 ? Math.floor(score) : '< 1'}
+            </EuiHealth>
+          );
+        }
+      },
+      truncateText: true,
+      width: '150px',
+    },
+    {
+      field: AnomalyDetectionListColumns.jobIds,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableNumJobs', {
+        defaultMessage: 'Jobs in group',
+      }),
+      render: (jobIds: Group['jobIds']) => jobIds.length,
+      sortable: true,
+      truncateText: true,
+      width: '100px',
+    },
+    {
+      field: AnomalyDetectionListColumns.latestTimestamp,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableLatestTimestamp', {
+        defaultMessage: 'Latest timestamp',
+      }),
+      dataType: 'date',
+      render: (time: number) => formatHumanReadableDateTimeSeconds(time),
+      textOnly: true,
+      sortable: true,
+      width: '20%',
+    },
+    {
+      field: AnomalyDetectionListColumns.docsProcessed,
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableDocsProcessed', {
+        defaultMessage: 'Docs processed',
+      }),
+      render: (docs: number) => toLocaleString(docs),
+      textOnly: true,
+      sortable: true,
+      width: '20%',
+    },
+    {
+      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableActionLabel', {
+        defaultMessage: 'Actions',
```

Comment:
The two Actions columns have different horizontal alignments. Could they both get the same?

![image](https://user-images.githubusercontent.com/7405507/65153753-3259cd00-da22-11e9-9b30-ce9dfc7c5728.png)

Reply:
Added to follow-up meta issue